packages/remark-custom-blocks/README.md

```diff
@@ -64,6 +64,11 @@ unified()
       classes: 'qux-block',
       title: 'required'
     },
+    slr: {
```

Comment:
It took me a while to figure out what `slr` was standing for. Please make it more explicit, a README should provide obvious examples. 😉

Reply:
Ok, it totaly makes sence :+1: 

It's juste that others custom-bock's names was 3 characters long so i tried to do the same.

Reply:
The current example blocks aren't named with a semantic in mind, they are [metasyntactic variables](https://en.wikipedia.org/wiki/Metasyntactic_variable). :)

Reply:
Can i add a commit to this PR to edit that ? Or i must do another PR -_-"

Reply:
Add this change to the existing commit and change the commit message to prefix it with `remark-custom-block: ` (see other commits: https://github.com/zestedesavoir/zmarkdown/commits/master?before=ba58ce66d5e7dd185e32c642a0503d1a2d583426+70 )